pyflagser/flagser.py

```diff
@@ -143,13 +151,17 @@ def flagser_weighted(adjacency_matrix, max_edge_weight=None, min_dimension=0,
 
     directed : bool, optional, default: ``True``
         If ``True``, computes persistent homology for the directed filtered
-        flag complex determined by `adjacency_matrix`. If False, computes
+        flag complex determined by `adjacency_matrix`. If ``False``, computes
         persistent homology for the undirected filtered flag complex obtained
-        by considering all weighted edges as undirected, and it is therefore
-        sufficient (but not necessary) to pass an upper-triangular matrix. When
-        ``False``, if two directed edges corresponding to the same undirected
-        edge are assigned different weights, only the one on the upper
-        triangular part of the adjacency matrix is considered.
+        by considering all weighted edges as undirected, and if two directed
```

Comment:
As you explained to me in https://github.com/giotto-ai/giotto-tda/pull/339#discussion_r430319243, it is not strictly true that:
> if two directed edges corresponding to the same undirected edge are assigned different weights, only the one on the upper triangular part of the adjacency matrix is considered

because the one in the upper triangular must also be less than or equal to `max_edge_weight` for this to be true.  How about this instead, since I guess we are assuming the readership here can cope with the extra detail:
> if two directed edges corresponding to the same undirected edge are explicitly assigned different weights and neither exceeds `max_edge_weight`, only the one in the upper triangular part of the adjacency matrix is considered

where I have also added the word "explicitly" to be more explicit about what we mean in the sparse case.

Reply:
Perfect!

Reply:
Great! I guess this should also be done for `flagser_count_weighted`.